### PR TITLE
Fixed deprecated method

### PR DESCRIPTION
**Pull request by marjasi**

### CODE REVIEW

**Comment by marjasi** [comment]
Don't know why there are 26,418 deletions, I guess Visual Studio deleted some unnecessary stuff, I didn't really mess with it in any way.

File: Mobile/ShopLens/Droid/TextVoicerActivity.cs [hunk 1]
```diff
@@ -39,7 +39,7 @@ private void SpeakOut()
         {
             string text = editTextToVoice.Text;
             if (!String.IsNullOrEmpty(text))
-                tts.Speak(text, QueueMode.Flush, null);
+                tts.Speak(text, QueueMode.Flush, null, null);
```
Comment (marjasi) [inline Mobile/ShopLens/Droid/TextVoicerActivity.cs:42]: This is the only thing that I did.

File: Mobile/ShopLens/Droid/SpeechActivity.cs [hunk 2]
```diff
@@ -101,6 +101,7 @@ protected override void OnActivityResult(int requestCode, Result resultCode, Int
                     }
                     // change the text back on the button
                     recButton.Text = "Start Recording";
+                    isRecording = false;
```
Comment (marjasi) [inline Mobile/ShopLens/Droid/SpeechActivity.cs:104]: Now you don't have to click two times to start the voice recognizer again. At least on my device you don't.

**Comment by EdvardasDlugauskas** [comment]
> 
> 
> Don't know why there are 26,418 deletions, I guess Visual Studio deleted some unnecessary stuff, I didn't really mess with it in any way.

Looks like you deleted \obj\ folders, which is good. They shouldn't be committed.

**Comment by mrjslau** [comment]
Nice. Works for me

**Pull request merged**